src/main/kotlin/org/datepollsystems/waiterrobot/mediator/printer/service/PrinterService.kt

```diff
@@ -52,6 +55,17 @@ object PrinterService {
             print(it.body.id, it.body.printerId, it.body.file.data)
         }
     }
+
+    fun printNetworkDisconnect() {
+        backendIdToPairing.values.forEach {
+            it.loPrinter.printPdf(
+                @Suppress("MagicNumber")
+                """Network_Disconnect_${Random.nextBytes(5).toHex()}""",
```

Comment:
```suggestion
                "Network_Disconnect_${Random.nextBytes(5).toHex()}",
```
String interpolation also works with regular Strings in Kotlin :)